coresdk/src/coresdk/animations.cpp

```diff
@@ -854,4 +854,10 @@ namespace splashkit_lib
             anim->entered_frame  = false;
         }
     }
+
+    animation create_animation(const string &script_name, int idx)
```

Comment:
No, this is the wrong fix for this. This function needs to be deleted from the header, not added to the implementation.